recipes/hdf5storage/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hdf5storage" %}
+{% set version = "0.1.14" %}
+{% set sha256 = "180593f91325c952346010dde32b359563f69d2760b6e7d25a8d2d851b93b1ab" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.zip
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - h5py >=2.3
+    - numpy
+    - scipy
```

Comment:
This is an [optional requirement]( https://github.com/frejanordsiek/hdf5storage/tree/0.1.14#dependencies ), but SciPy is necessary to support the [new and old MATLAB file formats]( https://github.com/frejanordsiek/hdf5storage/tree/0.1.14#matlab-mat-v73-file-support ) (SciPy supports the old one), which is still quite common.